distribution/std-lib/Base/src/Time/Duration.enso

```diff
@@ -0,0 +1,179 @@
+from Base import all
+import Base.Time.Time
+
+polyglot java import java.time.Duration as Java_Duration
+polyglot java import java.time.Period as Java_Period
+
+type Duration
+
+    ## An amount of time in terms of years, months, days, seconds and
+       nanoseconds.
+    type Duration interval_period interval_duration
+
+    ## Add specified amount of time to this duration.
+
+       > Example
+         Add 1 minute to a duration of 12 hours
+             12.hours . plus 1.minute
+
+       > Example
+         Add 1 hour to a duration of month.
+             1.month . plus 1.hour
+    plus : Duration -> Duration
+    plus other = Duration (this.interval_period . plus [other.interval_period]) (this.interval_duration . plus [other.interval_duration])
+
+    ## Subtract specified amount of time from this duration.
+       > Example
+         Subtract 1 nanosecond from a duration of 120 nanoseconds.
+             120.nano . minus 1.nano
+
+       > Example
+         Substract 3 hours from a duration of 12 years.
+             12.years . minus 3.hours
+    minus : Duration -> Duration
+    minus other = Duration (this.interval_period . minus [other.interval_period]) (this.interval_duration . minus [other.interval_duration])
+
+    ## Get the amount of nanoseconds of this duration.
+    get_nanos : Integer
```

Comment:
Remove `get` from the getters. Also use full name, i.e. `nanos` -> `nanoseconds`, `millis` -> `milliseconds`.